lib/resources/r4/condition.yaml

```diff
@@ -117,3 +184,27 @@ fields:
       - display: ICD-10-CM
         system: http://hl7.org/fhir/sid/icd-10-cm
         info_link: https://hl7.org/fhir/icd.html
+
+- name: onsetDateTime
```

Comment:
the field is onset, not onsetDateTime. same with abatement.

Reply:
Changed.
onset [here](https://github.com/nishanthnamana/fhir.cerner.com/commit/91fee55a512ed5f620c71861b3940e8479c3f61a#diff-abb99d2139d79f8a2a8f1d214f5f6882R188).
abatement [here](https://github.com/nishanthnamana/fhir.cerner.com/commit/91fee55a512ed5f620c71861b3940e8479c3f61a#diff-abb99d2139d79f8a2a8f1d214f5f6882R199).